김진/231214/신규-아이디-추천.js

```diff
@@ -0,0 +1,18 @@
+function solution(new_id) {
+  new_id = new_id
+    .toLowerCase(/[A-Z]/g)
```

Comment:
```suggestion
    .toLowerCase()
```
toLowerCase는 딱히 인수를 넘기지 않아도 모든 대문자를 소문자로 만들어 줍니다.